### PR TITLE
Proj1-1

### PR DESCRIPTION
**Pull request by GeekMind00**

### CODE REVIEW

File: lib/tasks/get_rates.rake [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'httparty'
```
Comment (HassanFathi100) [inline lib/tasks/get_rates.rake:3]: its time to partyyyy

Comment (GeekMind00) [reply lib/tasks/get_rates.rake:3]: yeeeeah!

File: lib/tasks/get_rates.rake [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+task get_rates: :environment do
+  response = HTTParty.get('https://api.apilayer.com/exchangerates_data/latest?apikey=vV1XjZwlnpEbzI3tIA5Bw2eqrg7631QJ')
```
Comment (HassanFathi100) [inline lib/tasks/get_rates.rake:6]: Nice, you solved the access key problem

File: lib/tasks/get_rates.rake [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  rates = response['rates']
+
+  rates.each do |name, value|
+    CurrencyRate.create!(name: name, value: value)
```
Comment (HassanFathi100) [inline lib/tasks/get_rates.rake:10]: change variables names
`name` -> `currencyName`
`value` -> `currencyValue`

Comment (GeekMind00) [reply lib/tasks/get_rates.rake:10]: Roger that.

File: db/migrate/20220604172321_create_currency_rates.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+class CreateCurrencyRates < ActiveRecord::Migration[7.0]
+  def change
+    create_table :currency_rates do |t|
+      t.string :name
```
Comment (HassanFathi100) [inline db/migrate/20220604172321_create_currency_rates.rb:4]: change this too
`name` -> `currencyName`
`value` -> `currencyValue`

File: lib/tasks/get_rates.rake (hunk 2 above)
Comment (HassanFathi100) [inline lib/tasks/get_rates.rake:6]: API key should be stored in secrets file!

Comment (GeekMind00) [reply lib/tasks/get_rates.rake:6]: Ouch, on it!

**Pull request merged**